horizon/opal_relay_api.py

```diff
@@ -59,7 +59,7 @@ class PDPPingRequest(BaseModel):
     platform: PDPPingPlatformState
 
 
-MAX_JWT_EXPIRY_BUFFER_TIME = 10
+MAX_JWT_EXPIRY_BUFFER_TIME = 60  # has to be more than the ping interval
```

Comment:
this wouldn't really fix the issue because the ping endpoint doesn't validate the JWT, it just sends it to the queue and continues, so the PDP actually never has knowledge of having an expired JWT

Reply:
@omer9564 there is a code in the PDP that checks this on every ping report.

Reply:
Oh my bad 😣